### PR TITLE
Add new size tokens

### PR DESCRIPTION
**Pull request by libertymayc**
Added size tokens

```diff
+ --salt-size-adornment
+ --salt-size-bar
+ --salt-size-selectable
+ --salt-size-icon
```

`--salt-size-base ` changed from a calculation on the basis unit to hardcoded values per density
`--salt-size-border` defined as 1px per density rather than set globally

### CODE REVIEW

File: .changeset/red-avocados-talk.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+Added size tokens
+
+```diff
++ --salt-size-bar
++ --salt-size-selectable
++ --salt-size-icon
+```
+
+`--salt-size-base ` changed from a calculation on the basis unit to hardcoded values per density
```
Comment (libertymayc) [inline .changeset/red-avocados-talk.md:13]: potential breaking change - unsure how to handle

**Comment by libertymayc** [comment]
drafted pending meeting 11/5

File: .changeset/red-avocados-talk.md (hunk 1 above)
Comment (joshwooding) [reply .changeset/red-avocados-talk.md:14]: This wouldn't be considered a breaking change since the API is the same

**Review by joshwooding — APPROVED** [review]

File: packages/lab/src/form-field-next/FormFieldNext.css [hunk 2]
```diff
@@ -3,7 +3,6 @@
   position: relative;
   column-gap: var(--salt-size-adornmentGap);
   row-gap: var(--salt-size-adornmentGap);
-  vertical-align: top;
```
Comment (origami-z) [inline packages/lab/src/form-field-next/FormFieldNext.css:6]: This line sneaked in .?

**Pull request merged**